scripts/index.js

```diff
@@ -1 +1,367 @@
+        return item.name;
+    } 
+};
+// CREATING HELPER FUNCTION(Creates list of only object NAMES with temps<70)^
+
+
+const arrayOfCoolCities = cities
+                    .filter(removeCitiesHotterThan70)
+                    .map(getCitiesCoolerThan70);
+// CALLING MAP & FILTER METHOD ^
+
+
+console.log(`#4`)
+console.log(arrayOfCoolCities);
+
+
+
+
+
+// 5. Cities 2
+// --------------
+    // Write a function which takes an array of city objects like the above problem as input and returns an array of the cities names.
+//________________________________________________________________________________________________________________________________
+
+function getAllCities(item) {
+  return item.name;
+};
+// CREATING HELPER FUNCTION(Determines if number isEven)^
+
+const arrayOfAllCities = cities.map(getAllCities);
+// CALLING MAP METHOD^
+
+console.log(`#5`)
+console.log(arrayOfAllCities);
+
+
+
+
+
+// 6. Good Job!
+// --------------
+    // Given an array of people's names:
+
+        const people = [
+          'Dom',
+          'Lyn',
+          'Kirk',
+          'Autumn',
+          'Trista',
+          'Jesslyn',
+          'Kevin',
+          'John',
+          'Eli',
+          'Juan',
+          'Robert',
+          'Keyur',
+          'Jason',
+          'Che',
+          'Ben'
+        ];
+    // Print out 'Good Job, {{name}}!' for each person's name, one on a line.
+//________________________________________________________________________________________________________________________________
+
+function printGoodJob(name) {
+    console.log(`Good Job, ${name}!`);
+}
+ 
+console.log(`#6`)
+people.map(printGoodJob);
+
+
+
+
+
+// 7. 3 times
+// --------------
+    // Given this function:
+
+        function call3Times(fun) {
+          fun();
+          fun();
+          fun();
+        };
+    // Use the call3Times function to print "Hello, world!" 3 times.
+//  ________________________________________________________________________________________________________________________________
+
+function fun() {
+    return console.log(`Hello World!!`);
+    
+}
+
+console.log(`#7`)
+call3Times(fun);
+
+
+
+
+
+//8. n times
+// --------------
+    // You will write a function callNTimes that takes two arguments: times as a number, and fun as a function. It will call that function for that many times. Example:
+
+    // > callNTimes(5, hello)
+    // Hello, world!
+    // Hello, world!
+    // Hello, world!
+    // Hello, world!
+    // Hello, world!
+    // You are allowed to use a loop in the implementation of callNTimes.
+//________________________________________________________________________________________________________________________________
+
+function hello(frequency) {
+    while (frequency > 0) {
+        console.log(`Hello World!`);
+        frequency--;
+    }
+};
+// CREATING HELPER FUNCTION (Is used in callNTimes)^
+
+function callNTimes(frequency, fun) {
+    fun(frequency);
```

Comment:
I wonder if the loop should be moved from `hello` to `callNTimes`?
That way, you can use `callNTimes` with any function, even ones that don't know to accept a `frequency` argument.